site/content/en/docs/api_sdk/sdk/lowlevel-api.md

```diff
@@ -641,3 +642,70 @@ with ApiClient(
     pprint(files) # ['sub/image_1.jpg', 'image_2.jpg']
 
 ```
+
+### Sending custom requests
+
+In some cases, you might need to send a custom request while using the `ApiClient`.
+Typically, it's desirable to make this request using the same configuration
+and authentication parameters as regular requests sent via the `ApiClient` instance.
+
+One particularly useful example for this is dataset export. When exporting a dataset,
+you receive a download URL from the server after the file is prepared
+(see example in [receiving data](#receiving-data)). This URL requires authentication,
+but it can't be made via the regular `ApiClient` endpoint methods.
+
+There are several options to make such a custom request via an `ApiClient` instance:
+- use `api_client.call_api()`
+- use `api_client.request()` or `api_client.rest_client` methods
```

Comment:
This mentions `rest_client`, but there's no details or examples of this in the text.

Reply:
Updated this section with a custom backend instead, seems to be more practical.